example/src/main/kotlin/com.expedia.graphql.sample/model/UnionModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.expedia.graphql.sample.model
+
+interface AUnionType
```

Comment:
do we want to rename it to something like `BodyPart` (`AUnionType` is pretty weird)?